lib/cisco_node_utils/command_reference_common.yaml

```diff
@@ -633,6 +633,19 @@ radius_server:
     config_set: '<state> radius-server host <ip> key <key>'
     default_value: null
 
+radius_server_group:
+  groups:
+    config_get: "show running-config radius all"
+    config_get_token: '/^aaa group server radius (.*) $/'
+    config_set: '<state> aaa group server radius <name>'
+    default_value: ''
+
+  hosts:
+    config_get: "show running-config radius all"
+    config_get_token: ['/^aaa group server radius %s $/i', '/server ((?:[0-9]{1,3}\.){3}[0-9]{1,3})/']
```

Comment:
Does not look like this will match an ipv6 server?   Can you check and add test cases for ipv6?

```
irb(main):035:0* ipv6_server = 'server 5:5::5:5'
=> "server 5:5::5:5"
irb(main):036:0> ipv4_server = 'server 192.168.0.1'
=> "server 192.168.0.1"
irb(main):039:0> /server ((?:[0-9]{1,3}\.){3}[0-9]{1,3})/.match(ipv4_server)
=> #<MatchData "server 192.168.0.1" 1:"192.168.0.1">
irb(main):040:0> /server ((?:[0-9]{1,3}\.){3}[0-9]{1,3})/.match(ipv6_server)
=> nil
irb(main):041:0> ipv6_server = 'server 2040::1'
=> "server 2040::1"
irb(main):042:0> ipv6_server = 'server 2040::1'
=> "server 2040::1"
irb(main):043:0> /server ((?:[0-9]{1,3}\.){3}[0-9]{1,3})/.match(ipv6_server)
=> nil
irb(main):044:0> 
```